resources/helm/v2.1/gateways/istio-ingress/templates/service.yaml

```diff
@@ -3,8 +3,8 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ $gateway.name | default "istio-ingressgateway" }}
-  namespace: {{ $gateway.namespace | default .Release.Namespace }}
+  name: {{ $gateway.name }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
This needs to be reverted, or it will break installation of gateways in other namespaces.